great_expectations/checkpoint/actions.py

```diff
@@ -1032,32 +1033,30 @@ def _run(
         )
 
 
+@public_api
+@deprecated_argument(
+    argument_name="target_site_names",
+    version="0.10.10",
+    message="target_site_names is deprecated as of v0.10.10 and will be removed in v0.16. Please use site_names instead.",
+)
 class UpdateDataDocsAction(ValidationAction):
-    """
-    UpdateDataDocsAction is a validation action that
-    notifies the site builders of all the data docs sites of the Data Context
-    that a validation result should be added to the data docs.
-
-    **Configuration**
-
-    .. code-block:: yaml
-
-        - name: update_data_docs
-        action:
-          class_name: UpdateDataDocsAction
+    """Notify the site builders of all data docs sites of a Data Context that a validation result should be added to the data docs.
 
     You can also instruct ``UpdateDataDocsAction`` to build only certain sites by providing a ``site_names`` key with a
-    list of sites to update:
-
-        - name: update_data_docs
-        action:
-          class_name: UpdateDataDocsAction
-          site_names:
-            - production_site
+    list of sites to update.
 
+    Args:
+        data_context: Data Context that is used by the Action.
+        site_names: Optional. A list of the names of sites to update.
+        target_site_names: Optional. Deprecated. A list of the names of sites to update.
```

Comment:
Sorry for the nit, just suggesting removing the Optional since it is included in the method signature and the Deprecated since you added the decorator:
```suggestion
        site_names: A list of the names of sites to update.
        target_site_names: A list of the names of sites to update.
```